### PR TITLE
Update psql version to 13

### PR DESCRIPTION
**Pull request by NikitaGandhi**
match psql client version same as postgres aurora database server version

### CODE REVIEW

**Review by alexgleith — APPROVED** [review]

File: index/Dockerfile [hunk 1]
```diff
@@ -25,6 +24,10 @@ RUN apt-get update \
     apt-get autoremove && \
     rm -rf /var/lib/{apt,dpkg,cache,log}
 
+# Install PostgreSQL Client 13
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
```
Comment (alexgleith) [inline index/Dockerfile:28]: Can we do this line about the other apt installs please, and install postgres as part of that still.

File: index/Dockerfile [hunk 2]
```diff
@@ -12,7 +12,6 @@ RUN apt-get update \
     python3-dev \
     # For Psycopg2
     libpq-dev\
-    postgresql-client-12 \
```
Comment (alexgleith) [inline index/Dockerfile:15]: Let's still install postgres here, please :-)

File: index/Dockerfile (hunk 1 above)
Comment (NikitaGandhi) [reply index/Dockerfile:30]: > Can we do this line about the other apt installs please, and install postgres as part of that still.

@alexgleith there are some dependancy packages needed (e.g. wget, lsb-release) to configure postgres repository and key so can't install it in the same place

Comment (alexgleith) [reply index/Dockerfile:30]: Ah ok... Then what we want to do is this:

```
    && apt-get autoclean \
    && apt-get autoremove \
    && rm -rf /var/lib/{apt,dpkg,cache,log}
```

after the apt commands. This keeps the image smaller.

Comment (NikitaGandhi) [reply index/Dockerfile:30]: done

**Review by alexgleith — APPROVED** [review]

**Pull request merged**